### PR TITLE
Fixed ParseError in stub file

### PR DESCRIPTION
**Pull request by GrahamCampbell**
`bool` cannot have a default value of an integer. I think this issue probably exists in the C code too.

### CODE REVIEW

File: php_memcached.stub.php [hunk 1]
```diff
@@ -17,8 +17,8 @@ public function get(string $key, callable $cache_cb=NULL, int $get_flags=0): mix
 	public function getByKey(string $server_key, string $key, callable $cache_cb=NULL, int $get_flags=0): mixed {}
 	public function getMulti(array $keys, int $get_flags=0): false|array {}
 	public function getMultiByKey(string $server_key, array $keys, int $get_flags=0): false|array {}
-	public function getDelayed(array $keys, bool $with_cas=0, callable $value_cb=NULL): bool {}
-	public function getDelayedByKey(string $server_key, array $keys, bool $with_cas=0, callable $value_cb=NULL): bool {}
+	public function getDelayed(array $keys, bool $with_cas=NULL, callable $value_cb=NULL): bool {}
+	public function getDelayedByKey(string $server_key, array $keys, bool $with_cas=NULL, callable $value_cb=NULL): bool {}
```
Comment (sodabrew) [inline php_memcached.stub.php:21]: Seems like `false` would be the correct value then?
https://www.php.net/manual/en/language.types.boolean.php#language.types.boolean.casting

Comment (GrahamCampbell) [reply php_memcached.stub.php:21]: You can have a `bool` variable with a `null` default return value. PHP internally changes the type to `bool|null`, the same as it does with the `callable` typehints in this file. The PHP 7 version of the stub file has `null` as the default.

Comment (GrahamCampbell) [reply php_memcached.stub.php:21]: PHP will not cast `null` to a bool here. It will keep it as `null`.

Comment (GrahamCampbell) [reply php_memcached.stub.php:21]: https://3v4l.org/RlWJe

Comment (sodabrew) [reply php_memcached.stub.php:21]: Oh, I see you had FALSE as one of the intermediate commits.

**Comment by GrahamCampbell** [comment]
It looks like the issue still persits. I guess the C code needs fixing too?

**Comment by GrahamCampbell** [comment]
` php_memcached_arginfo.h` needs to be re-generated.

**Pull request merged**